src/itemdrop.zig

```diff
@@ -116,19 +118,17 @@ pub const ItemDropManager = struct { // MARK: ItemDropManager
 	}
 
 	pub fn getPositionAndVelocityData(self: *ItemDropManager, allocator: NeverFailingAllocator) []u8 {
-		const _data = allocator.alloc(u8, self.size*50);
-		var data = _data;
+		var writer = utils.BinaryWriter.initCapacity(allocator, .big, self.size*50);
```

Comment:
Should be main.network.networkEndian, since this is sent over the network.

Reply:
```suggestion
		var writer = utils.BinaryWriter.initCapacity(allocator, main.network.networkEndian, self.size*50);
```